.bazelrc

```diff
@@ -41,3 +22,75 @@ build:fastdbg -c fastbuild
 build:fastdbg --cxxopt='-gline-tables-only' --host_cxxopt='-gline-tables-only'
 build:fastdbg --linkopt='-gline-tables-only' --host_linkopt='-gline-tables-only'
 build:fastdbg --strip=never
+
+#
+# Linux
+#
+build:linux --cxxopt='-std=c++20' --host_cxxopt='-std=c++20' --force_pic --verbose_failures
+build:linux --cxxopt='-fcoroutines-ts' --host_cxxopt='-fcoroutines-ts'
+build:linux --cxxopt='-stdlib=libc++' --host_cxxopt='-stdlib=libc++'
+build:linux --linkopt='-stdlib=libc++' --host_linkopt='-stdlib=libc++'
+
+build:linux --action_env=BAZEL_COMPILER=clang
+build:linux --action_env=CC=clang
+build:linux --action_env=CXX=clang++
+
+# Warning options.
+build:linux --cxxopt='-Wall' --host_cxxopt='-Wall'
+build:linux --cxxopt='-Wextra' --host_cxxopt='-Wextra'
+build:linux --cxxopt='-Wno-strict-aliasing' --host_cxxopt='-Wno-strict-aliasing'
+build:linux --cxxopt='-Wno-sign-compare' --host_cxxopt='-Wno-sign-compare'
+build:linux --cxxopt='-Wno-unused-parameter' --host_cxxopt='-Wno-unused-parameter'
+build:linux --cxxopt='-Wno-missing-field-initializers' --host_cxxopt='-Wno-missing-field-initializers'
+build:linux --cxxopt='-Wno-ignored-qualifiers' --host_cxxopt='-Wno-ignored-qualifiers'
+
+build:linux --@capnp-cpp//src/kj:openssl=True --@capnp-cpp//src/kj:zlib=True --@capnp-cpp//src/kj:libdl=True
+
+#
+# macOS
```

Comment:
To avoid duplication, how about defining e.g. `build:unix`, assigning the options to it and using `build:macos -c unix`, `build:linux -c unix` to have them for both systems?